### PR TITLE
get subgraph no prop

### PR DESCRIPTION
**Pull request by nevermore3**

### CODE REVIEW

File: tests/tck/features/subgraph/subgraph.IntVid.feature [hunk 1]
```diff
@@ -10,86 +10,86 @@ Feature: Integer Vid subgraph
   Scenario: Integer Vid invalid input
     When executing query:
       """
-      GET SUBGRAPH FROM $-.id
+      GET SUBGRAPH WITH PROP FROM $-.id
```
Comment (Shylock-Hg) [inline tests/tck/features/subgraph/subgraph.IntVid.feature:13]: It's incompatible. Why don't try `without prop`?

Comment (nevermore3) [reply tests/tck/features/subgraph/subgraph.IntVid.feature:13]: To be consistent with the path sentence

**Review by yixinglu — DISMISSED** [review]

File: src/validator/GetSubgraphValidator.cpp [hunk 2]
```diff
@@ -241,16 +245,20 @@ Status GetSubgraphValidator::toPlan() {
     return Status::OK();
 }
 
-StatusOr<std::vector<storage::cpp2::VertexProp>> GetSubgraphValidator::buildVertexProp() {
+StatusOr<GetNeighbors::VertexProps> GetSubgraphValidator::buildVertexProp() {
```
Comment (Shylock-Hg) [inline src/validator/GetSubgraphValidator.cpp:248]: I think you could reuse `SchemaUtil::genVertexProp` function.

Comment (nevermore3) [reply src/validator/GetSubgraphValidator.cpp:248]: Will be refactored later, move  it to planner

**Comment by laura-ding** [comment]
has AddressSanitizer error, solve it first.

**Review by laura-ding — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**